Fix API reference in docs for setting instrumentation key

JonathanGiles opened the pull request:
The docs appear to reference the C# API rather than the Java API. However, the URL linked above the code snippet - https://docs.microsoft.com/en-us/azure/application-insights/app-insights-api-custom-events-metrics#ikey - does not even have a Java section, and I recommend that this be rectified as well.

azurecla commented:
Hi __@ostrich75__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (larryzh). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

ostrich75 commented:
#sign-off

JonathanGiles committed:
Fix API reference in docs for setting instrumentation key

The docs appear to reference the C# API rather than the Java API. However, the URL linked above the code snippet - https://docs.microsoft.com/en-us/azure/application-insights/app-insights-api-custom-events-metrics#ikey - does not even have a Java section, and I recommend that this be rectified as well.
